api/HTMLImageElement.json

```diff
@@ -813,7 +813,7 @@
               "version_added": true
             },
             "firefox_android": {
-              "version_added": null
+              "version_added": true
```

Comment:
Is this just mirroring the desktop data, or how do we know `lowsrc` is supported on current Firefox for Android?

Reply:
The collector has results for Firefox Android 83, and is updating Fenix data with the results -- so, this is confirmed supported on Fenix!

Reply:
OK, excellent!